synadm/cli/room.py

```diff
@@ -315,3 +315,27 @@ def make_admin(helper, room_id, user_id):
     mxid = helper.generate_mxid(user_id)
     out = helper.api.room_make_admin(room_id, mxid)
     helper.output(out)
+
+
+@room.command()
+@click.argument("room_id", type=str)
+@click.option(
+        "--block/--unblock", "-b", type=bool, default=True,
+        help="Specifies whether to block or unblock a room."
+)
```

Comment:
It's not 100% obvious that there is a default set and what it actually is when looking at the help of that command:
```
(synadm) ~/git/jacksonchen666_synadm (room-blocking) $ synadm room block
Usage: synadm room block [OPTIONS] ROOM_ID
Try 'synadm room block -h' for help.

Error: Missing argument 'ROOM_ID'.
(synadm) ~/git/jacksonchen666_synadm (room-blocking) $ synadm room block -h
Usage: synadm room block [OPTIONS] ROOM_ID

  Block or unblock a room.

Options:
  -b, --block / --unblock  Specifies whether to block or unblock a room.
  -h, --help               Show this message and exit.
(synadm) ~/git/jacksonchen666_synadm (room-blocking) $ 
```

You might want to set show_defaults=True